terraform/aws-users.tf

```diff
@@ -0,0 +1,21 @@
+module "iam_user_gwenstacy" {
+  source = "./modules/users"
+
+  user_name = "gwenstacy"
+  user_tags = {
+    "Environment" = "Development"
+    "Project"     = "spiderverse"
+  }
+  pgp_key = "user_provided_public_key_here"
```

Comment:
My thoughts are that users should create their own keys (will provide instructions, of course). Then they can provide us their public key, which will be used to encrypt their password, and then we can provide them the encrypted password...also forcing a pw change on first login. Thoughts? I'm not exactly sure how to handle the output of the encrypted pw...I _think_ it would need to be visible in the output for us to capture it

Reply:
took out the pgp key. i think for now we can create the user via terraform and then a part of the flow will be that once that's done, someone with access to reset passwords can generate a pw in 1pw and run a quick command via the cli to trigger a reset using the pw. does that seem reasonable for now? it creates one extra step, but no need to click around in the ui